Copyediting of descriptions for Advanced Node & Express challenges (#39606)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/authentication-strategies.english.md

## Description
<section id='description'>

A strategy is a way of authenticating a user. You can use a strategy for allowing users to authenticate based on locally saved information (if you have them register first) or from a variety of providers such as Google or GitHub. For this project, we will set up a local strategy. To see a list of the hundreds of strategies, visit Passport's site <a href='http://passportjs.org/' target='_blank'>here</a>.

> Add <em>passport-local</em> as a dependency and add it to your server as follows: <code>const LocalStrategy = require('passport-local');</code>

Reviewer: ```suggestion
Add <code>passport-local</code> as a dependency and add it to your server as follows: <code>const LocalStrategy = require('passport-local');</code>
```

> Now you will have to tell passport to <b>use</b> an instantiated LocalStrategy object with a few settings defined. Make sure this (as well as everything from this point on) is encapsulated in the database connection since it relies on it!

Reviewer 2: ```suggestion
Now you will have to tell passport to <b>use</b> an instantiated <code>LocalStrategy</code> object with a few settings defined. Make sure this (as well as everything from this point on) is encapsulated in the database connection since it relies on it!
```

));
```

> This is defining the process to use when we try to authenticate someone locally. First, it tries to find a user in our database with the username entered, then it checks for the password to match, then finally, if no errors have popped up that we checked for, like an incorrect password, the user's object is returned and they are authenticated.

Reviewer: ```suggestion
This is defining the process to use when we try to authenticate someone locally. First, it tries to find a user in our database with the username entered, then it checks for the password to match, then finally, if no errors have popped up that we checked for, like an incorrect password, the <code>user</code>'s object is returned and they are authenticated.
```
